flask_smorest/blueprint.py

```diff
@@ -116,6 +117,8 @@ def add_url_rule(
             in this path, only used to document the resource.
         :param list tags: List of tags for the resource.
             If None, ``Blueprint`` name is used.
+        :param boolean hidden_from_api_spec: Hide documentation in swagger UI.
+            Default False.
```

Comment:
Not only Swagger-UI.

```py
        :param boolean hidden_from_api_spec: Don't document resource in API
            spec file.
            Default False.
```

Reply:
I've fixed this locally, how would you like me to commit it? I would like to avoid squashing commits if possible